docs/version0.15.md

```diff
@@ -70,6 +71,16 @@ For compatibility, the [`-XX:OnOutOfMemoryError`](xxonoutofmemoryerror.md) OpenJ
 
 The VM now supports the allocation of huge pages on Linux when you use the `madvise` (`/sys/kernel/mm/transparent_hugepage/enabled`) setting. To enable this feature, set [`-XX:+TransparentHugePage`](xxtransparenthugepage.md) on the command line when you start your application. This option is currently not enabled by default.
 
+###  ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) Support for low-overhead heap profiling
```

Comment:
Should there be an End of content tag?

Reply:
As it is on the title of the section I think it is ok. It is also consistent with other sections in that topic?

Reply:
It's not consistent with the ChaCha20 topic, which has the end mark.

Reply:
Erm ... so it is. And that's not consistent with other release topics either. OK - fixing to make the same in this topic.